### PR TITLE
x11-wm/qtile: fix xcffib.ffi_build errors

### PR DESCRIPTION
**Pull request by mrtnvgr**
- https://github.com/tych0/xcffib/commit/517169a2fb6dcc127059f581d3890e0861e2e950
- https://github.com/tych0/xcffib/issues/155
- https://github.com/qtile/qtile/pull/4289

### CODE REVIEW

File: x11-wm/qtile/qtile-0.22.1-r1.ebuild [hunk 1]
```diff
@@ -31,7 +31,7 @@ RDEPEND="
 	dev-python/dbus-next[${PYTHON_USEDEP}]
 	dev-python/pygobject[${PYTHON_USEDEP}]
 	>=dev-python/six-1.4.1[${PYTHON_USEDEP}]
-	>=dev-python/xcffib-0.10.1[${PYTHON_USEDEP}]
+	<dev-python/xcffib-1.4.0[${PYTHON_USEDEP}]
```
Comment (thesamesam) [inline x11-wm/qtile/qtile-0.22.1-r1.ebuild:34]: Can you add a comment above RDEPEND with a link to the upstream bug etc?

**Comment by thesamesam** [comment]
Thanks! I've pushed with a revbump given it looks like it might impact runtime as well.

**Comment by ftpd** [comment]
@mrtnvgr can we also have this for -9999 ebuild?

**Comment by mrtnvgr** [comment]
> @mrtnvgr can we also have this for -9999 ebuild?

The PR regarding this issue is almost done, so this will be fixed very soon.

**Pull request closed without merging**